File: Pascal/simple_login_5.pas
program simple_login_5;

var 
 username, password: string;

procedure simple_login(username:string; password:string);
 begin
  if (username = 'John') and (password = 'Doe') then
   writeln('Hello ', username,' ', password)
  else if (username = 'Mary') and (password = 'Jane') then
   writeln('Hello ', username,' ', password)
  else if (username = 'Fajar') and (password = 'Purnama') then
   writeln('Hello ', username,' ', password)
  else
   writeln('Wrong username or password!');
  
  writeln('Program terminate!'); 
 end;

begin
 write('username: '); readln(username);
 write('password: '); readln(password);
 simple_login(username, password);
end.

File: C/simple_login_5.c
#include <stdio.h>

void login(char username[20], char password[20]) {
 
 if ((strncmp(username, "John", 20) == 0) && (strncmp(password, "Doe", 20)) == 0) {
  printf("Hello %s %s.\n", username, password);
 } else if ((strncmp(username, "Mary", 20) == 0) && (strncmp(password, "Jane", 20) == 0)) {
  printf("Hello %s %s.\n", username, password);
 } else if ((strncmp(username, "Fajar", 20) == 0) && (strncmp(password, "Purnama", 20)) == 0) {
  printf("Hello %s %s.\n", username, password);
 } else {
  printf("Wrong username or password!\n");
 }
}

int main() {
 char username[20], password[20];

 printf("Username: "); scanf("%s", username);
 printf("Password: "); scanf("%s", password);
 login(username, password);
 return 0;
}
